hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,11 @@
+RELEASE_TYPE: minor
+
+This version adds support for generating numpy.ndarray and pandas.Series with any python object as an element.
+Effectively, hypothesis can now generate ``np.array([MyObject()], dtype=object)``.
+The first use-case for this is with Pandas and Pandera where it is possible and sometimes required to have columns which themselves contain structured datatypes.
+Pandera seems to be waiting for this change to support ``PythonDict, PythonTypedDict, PythonNamedTuple`` etc.
+
+---
+
+- Accept ``dtype.kind = 'O'`` in ``from_dtype``
+- Use ``.iat`` instead of ``.iloc`` to set values in pandas strategies
```

Comment:
(apologies for this comment, it's late at night & I don't really know what I want to do instead, but thought it better to send a review now than wait until later)

I'd like to rework this note, to focus more tightly on the specific changes - as prose, not dot-points - and then afterwards note why this is valuable, with pandera only mentioned as one possible case for structured data within a pandas series.  I'd also include cross-references to each class you mention, and (optional but encouraged) a thank-you note to yourself at the end of the changelog ("Thanks to Shaun Read for identifying and fixing these issues!" or similar).